lib/hyacinth/adapters/external_identifier_adapter/datacite/rest_api/v2/data.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+require 'json'
+class Hyacinth::Adapters::ExternalIdentifierAdapter::Datacite::RestApi::V2::Data
+  MINIMUM_REQUIRED_ATTRIBUTES_FOR_FINDABLE_DOI = [:title,
+                                                  :creators,
+                                                  :doi_url,
+                                                  :publisher,
+                                                  :publication_year,
+                                                  :resource_type,
+                                                  :doi_prefix].freeze
+  # Events used when minting a DOI, depends on desired state of minted DOI
+  # see https://support.datacite.org/docs/how-do-i-make-a-findable-doi-with-the-rest-api
+  DOI_STATES = [:draft, :findable, :registered].freeze
+  DOI_MINT_EVENT = { draft: '',
+                     findable: 'publish',
+                     registered: 'hide' }.freeze
+
+  attr_accessor :attributes,
+                :data_hash,
+                :creators,
+                :publisher,
+                :publication_year,
+                :resource_type_general, # controlled, usually "Text"
+                :title,
+                :url
+
+  attr_accessor :prefix
```

Comment:
Any reason not to combine this accessor with the other list above it?

Reply:
accessor has been collapsed into the generalized accessor list.